### PR TITLE
Merge release/5.0 to develop

### PR DESCRIPTION
**Pull request by rohitsinha54**
No conflicts

### CODE REVIEW

File: conf/autobuild.json [hunk 1]
```diff
@@ -5,13 +5,19 @@
         "local_user": "root",
         "repos": {
           "cask-tracker": {
-            "branch": "develop"
+            "branch": "release/0.7"
```
Comment (anwar6953) [inline conf/autobuild.json:8]: Undo these changes

**Comment by rohitsinha54** [comment]
Closing in favor of https://github.com/caskdata/cdap-integration-tests/pull/940
